docs/en/operation/Admin-CLI.md

```diff
@@ -300,3 +301,13 @@ $ ./bin/alluxio fsadmin ufs --mode readOnly hdfs://ns
 
 The `fsadmin ufs` subcommand takes a UFS URI as an argument. The argument should be a root
 UFS URI like `hdfs://<name-service>/`, and not `hdfs://<name-service>/<folder>`.
+
+### updateConf
+
+The `updateConf` command provides a way to update config for current running services, the request is sent to alluxio master directly,
+but the other services like worker, fuse, s3 proxy or some other clients will aware the config changed and sync the config.
+
+```console
+$ ./bin/alluxio fsadmin updateConf key1=val1 key2=val2
+Updated 2 configs
+```
```

Comment:
I assume this command don't have general support for dynamic configurations update. Can you also add the supported configuration list in this doc https://github.com/Alluxio/alluxio/pull/13643?

Reply:
Sure